src/integrationTest/java/uk/gov/hmcts/darts/common/controller/GetWelcomeTest.java

```diff
@@ -4,27 +4,20 @@
 import org.junit.jupiter.api.Test;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
-import org.springframework.boot.test.mock.mockito.MockBean;
 import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.ContextConfiguration;
 import org.springframework.test.web.servlet.MockMvc;
 import org.springframework.test.web.servlet.MvcResult;
-import uk.gov.hmcts.darts.audio.repository.AudioRequestRepository;
-import uk.gov.hmcts.darts.notification.repository.NotificationRepository;
 
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
 import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
 
 @WebMvcTest
```

Comment:
Wondering whether this should be 
`@WebMvcTest(RootController.class)`

Looking at https://spring.io/guides/gs/testing-web/
Then we probably don't need the `ContextConfiguration` line?